compressed_depth_image_transport/src/codec.cpp

```diff
@@ -0,0 +1,334 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2016 Google, Inc.
```

Comment:
We probably want to keep Willow in here too no ? It seems too easy to get copyright by just moving text around :)